libraries/tlv-account-resolution/src/pod.rs

```diff
@@ -61,6 +61,17 @@ impl PodAccountMeta {
             is_writable: is_writable.into(),
         })
     }
+
+    /// Returns `true` if this `PodAccountMeta` represents a standard
+    /// `AccountMeta`
+    pub fn is_account_meta(&self) -> bool {
+        self.discriminator == 0
+    }
+
+    /// Returns `true` if this `PodAccountMeta` represents a PDA
+    pub fn is_pda(&self) -> bool {
+        self.discriminator == 1
+    }
```

Comment:
The enum is cool but that forces you to import the enum as well as whatever else you're importing from TLV AR. Do you think that's annoying for people or nah? With this setup, it's just a boolean, and now that I think of it we can probably just have one, since the structure is binary

Reply:
It's not a huge deal either way, but it's true that just having one of the functions would be simpler than an enum.

On the other hand, it would probably be much clearer to move `resolve_account_meta` and `resolve_pda` into this struct and not have to worry about this function at all, since it's only used there.